gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -0,0 +1,45 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationPrimary < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize
+      set_defaults
+    end
+
+    def set_defaults
+      self.cluster_name              = "replication_cluster"
+      self.node_number               = 1
```

Comment:
I don't think we should offer a default for `node_number` if, when they select the defaults, we know it will fail (which in this case it will because the node number needs to be unique within the cluster).